doc/CLI-user-manual/README.md

```diff
@@ -139,28 +150,28 @@ $ kc show-wallet
 ### Backing Up and Recovering a Wallet
 
 To recover a wallet from a backup, you need two pieces of information:
-- the seed phrase, aka mnemonic
-- the backup DID
+- the seed phrase, aka mnemonic,
+- the backup DID.
 
-The mnemonic consists of 12 short words (BIP-39) that are used to generate the wallet's private keys
+The mnemonic consists of 12 short words (BIP-39) that are used to generate the wallet's private keys:
 
-```
+```sh
 $ kc show-mnemonic
 know soon mind pen polar pulse patient salmon wage friend equip rotate
 ```
 
-Creating a wallet backup encrypts the current state of a user wallet content in a DID Document.
+Creating a wallet backup encrypts the current state of a user wallet content in a DID Document:
 
-```
+```sh
 $ kc backup-wallet
 did:mdip:test:z3v8Auairrc7XjSdoA1QvuytZXmGdmjcaFsPb2xKjM6TzowPKRn
 ```
 
-If you lose the wallet file, you can regenerate the private keys from the mnemonic
+If you lose the wallet file, you can regenerate the private keys from the mnemonic:
 
-!!! Danger: import-wallet will overwrite an existing wallet!
+!!! danger import-wallet will overwrite an existing wallet!
```

Comment:
```suggestion
```

This was fixed, so this warning can be removed

Reply:
Removed in 7762b1350da82734ccf51d4431c5aefebd009a55